plugins/modules/packaging/os/pkgutil.py

```diff
@@ -11,46 +11,53 @@
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: pkgutil
 short_description: Manage CSW-Packages on Solaris
 description:
-    - Manages CSW packages (SVR4 format) on Solaris 10 and 11.
-    - These were the native packages on Solaris <= 10 and are available
-      as a legacy feature in Solaris 11.
-    - Pkgutil is an advanced packaging system, which resolves dependency on installation.
-      It is designed for CSW packages.
-author: "Alexander Winkler (@dermute)"
+- Manages CSW packages (SVR4 format) on Solaris 10 and 11.
+- These were the native packages on Solaris <= 10 and are available as a legacy feature in Solaris 11.
+- Pkgutil is an advanced packaging system, which resolves dependency on installation.
+  It is designed for CSW packages.
+version_added: '1.3'
+author:
+- Alexander Winkler (@dermute)
+- David Ponessa (@scathatheworm)
 options:
   name:
     description:
-      - Package name, e.g. (C(CSWnrpe))
+    - The name of the package.
+    - When using C(state=latest), this can be C('*'), which updates all installed packages managed by pkgutil.
+    type: list
     required: true
     type: str
   site:
     description:
-      - Specifies the repository path to install the package from.
-      - Its global definition is done in C(/etc/opt/csw/pkgutil.conf).
+    - The repository path to install the package from.
+    - Its global definition is done in C(/etc/opt/csw/pkgutil.conf).
     required: false
     type: str
   state:
     description:
-      - Whether to install (C(present)), or remove (C(absent)) a package.
-      - The upgrade (C(latest)) operation will update/install the package to the latest version available.
-      - "Note: The module has a limitation that (C(latest)) only works for one package, not lists of them."
-    required: true
-    choices: ["present", "absent", "latest"]
+    - Whether to install (C(present)), or remove (C(absent)) packages.
+    - The upgrade (C(latest)) operation will update/install the packages to the latest version available.
     type: str
+    choices: [ absent, installed, latest, present, removed ]
+    default: present
   update_catalog:
     description:
-      - If you want to refresh your catalog from the mirror, set this to (C(yes)).
-    required: false
+    - If you want to refresh your catalog from the mirror, set this to (C(yes)).
+    type: bool
     default: no
+  force:
+    description:
+    - Allows the update process to downgrade packages to what is present in the repostory set this to (C(yes)).
```

Comment:
```suggestion
    - Allows the update process to downgrade packages to what is present in the repository set this to C(yes).
```
I'm not really sure what this sentence means.